com.unity.render-pipelines.high-definition/Documentation~/Anti-Aliasing.md

```diff
@@ -43,6 +43,8 @@ To select TAA for a Camera:
 1. Select the Camera in the Scene view or Hierarchy and view it in the Inspector.
 2. In the General section, select Temporal Anti-aliasing (TAA) from the Anti-aliasing drop-down.
 
+When using the same Camera GameObject for multiple Game Views TAA may not work as expected due to limitations of the history buffer system. Multiple game views using different Cameras will however work as expected.
```

Comment:
maybe add this into a "limitatoin section" : cc @JordanL8 ?

Reply:
Yeah, a limitations section per anti-aliasing effect (when necessary) sounds good. For this one:

\### Limitations
In the Editor, if multiple Game views use the same Camera, TAA may not work as expected due to limitations of the history buffer system. However, if you use multiple Game views, where each Game view uses a unique Camera, TAA works as expected.